src/modules/activities/AdminDashboard/UsersActivity/UsersSkeleton.tsx

```diff
@@ -0,0 +1,13 @@
+import SkeletonElement from '@/modules/_partials/shared/SkeletonElement';
+
+const UsersSkeleton = () => {
+  return (
+    <div className="flex justify-between flex-col  bg-red-500 dark:bg-gray-800  p-2">
+      <SkeletonElement className="w-full h-16 rounded" />
+      <SkeletonElement className="w-full h-16 rounded " />
+      <SkeletonElement className="w-full h-16 rounded" />
```

Comment:
@bizip remove duplicates; Create an array that represents the number of skeleton elements you want to render. Use the map function to iterate over the array and render the `SkeletonElement` component for each iteration, providing a unique key and applying the common class names.